app/jobs/request-payment-status.js

```diff
@@ -49,7 +52,17 @@ const processPaymentDataEntry = async (paymentDataEntry, logger) => {
   if (status.name === PaymentHubStatus.SETTLED) {
     await processPaidClaim(claimReference, logger)
   } else {
-    await incrementPaymentCheckCount(claimReference)
+    const [affectedRows] = await incrementPaymentCheckCount(claimReference)
+
+    if (affectedRows?.[0]?.[0]?.paymentCheckCount === PAYMENT_CHECK_COUNT_LIMIT) {
```

Comment:
Do we need the triple guard here? Surely if there is an affected row we can assume the elements within will always conform to standard shape